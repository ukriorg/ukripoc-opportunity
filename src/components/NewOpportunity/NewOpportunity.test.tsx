import React from "react";
import renderer from "react-test-renderer";
import { NewOpportunity } from "./index";

describe("NewOpportunity", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<NewOpportunity addOpportunity={() => {}} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
