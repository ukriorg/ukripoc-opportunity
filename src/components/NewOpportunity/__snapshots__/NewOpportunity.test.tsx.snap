// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewOpportunity renders correctly 1`] = `
<div
  className="wrap"
>
  <h2
    className="sc-htpNat bpICJA"
    size="LARGE"
  >
    New opportunity
  </h2>
  <h4
    className="sc-htpNat ipcdBj"
    size="SMALL"
  >
    Opportunity name
  </h4>
  <span
    className="sc-bxivhb eBoyen"
  >
    This name will be used to identify the opportunity internally and publicly.
  </span>
  <details
    className="sc-ifAKCX cyIqaI"
    open={false}
  >
    <summary
      className="sc-EHOje fxpUSX"
    >
      <span
        className="sc-bZQynM HMsRn"
      >
        How should I name my opportunity?
      </span>
    </summary>
    <div
      className="sc-gzVnrw kSeSEc"
    >
      Give your opportunity a name that will help potential applicants understand what the opportunity is about. You don't need to add tags (such as the year) into opportunity names. Opportunities will be tagged automatically based on other information you input.
    </div>
  </details>
  <input
    className="sc-htoDjs kRSYaw"
    onChange={[Function]}
    type="text"
    value=""
  />
  <button
    className="sc-bdVaJa bBDdCP"
    disabled={false}
    onClick={[Function]}
  >
    Create opportunity
  </button>
</div>
`;
